helm-chart/README.md

```diff
@@ -81,3 +81,62 @@ Not all have IPV6 enabled, hence this has to be disabled as follows:
 helm install kubeshark kubeshark/kubeshark \
   --set tap.ipv6=false
 ```
+
+## Configuration
+
+| Parameter                                 | Description                                   | Default                                                 |
+|-------------------------------------------|-----------------------------------------------|---------------------------------------------------------|
+| `tap.docker.registry`                     | Docker registry URL                           | `docker.io/kubeshark`                                   |
+| `tap.docker.tag`                          | Docker image tag                              | `latest`                                                |
+| `tap.docker.imagePullPolicy`              | Kubernetes image pull policy                  | `Always`                                                |
+| `tap.docker.imagePullSecrets`             | Kubernetes secrets to pull Docker images      | `[]`                                                    |
+| `tap.proxy.worker.srvport`                | Worker service port                           | `8897`                                                  |
+| `tap.proxy.hub.port`                      | Hub service port                              | `8898`                                                  |
+| `tap.proxy.hub.srvport`                   | Hub service port (internal)                   | `8898`                                                  |
+| `tap.proxy.front.port`                    | Front-facing service port                     | `8899`                                                  |
+| `tap.proxy.host`                          | Allowed IPs                                   | `127.0.0.1`                                             |
+| `tap.namespaces`                          | List of namespaces to observe                 | `[]`                                                    |
+| `tap.release.repo`                        | URL for the Helm chart repository             | `https://helm.kubeshark.co`                             |
+| `tap.release.name`                        | Name for the release                          | `kubeshark`                                             |
+| `tap.release.namespace`                   | Namespace to install the chart                | `default`                                               |
+| `tap.persistentstorage`                   | Enable using persistent storage for the worker | `false`                                                |
```

Comment:
```suggestion
| `tap.persistentstorage`                   | Use `persistentVolumeClaim` instead of `emptyDir` | `false`                                                |
```